src/config/session-management.md

```diff
@@ -40,3 +40,13 @@ service, as waiting for a D-Bus activation can lead to issues.
 There is an alternative D-Bus configuration which takes advantage of elogind for
 features such as seat detection. It requires installing the `dbus-elogind`,
 `dbus-elogind-libs` and `dbus-elogind-x11` packages.
+
+## seatd
+
+[seatd(1)](https://man.voidlinux.org/seatd.1) is a minimal seat management
+daemon and an alternative to elogind.
+
+To use it, install the `seatd` package and enable its service. If you want
+non-root users to access the seatd session, add them to the `seatd` group.
```

Comment:
I think groups from packages should begin with an underscore, so it should be `_seatd`... This should be discussed in the PR, of course.

Reply:
Well, there are already several packages in the repos that provide groups without underscores, such as `chrony`, `socklog`, `weston` (which apparently doesn't delete its group after being removed from the system), and I am sure others... so if an underscore at the beginning truly is the norm, these should be fixed as well.

Reply:
Fixed on both sides (seatd PR & here).